main.py

```diff
@@ -65,6 +65,7 @@ def change_read():
         new_read = ui.get_read_value()
         book.read = new_read
         book.save()
+        ui.message('You have read ' + book.title + ' by ' + book.author)
```

Comment:
This doesn't check if the person is changing the status from read to unread, or vice versa. I tested the code and changed a books status from "read" to "unread", and it responded by saying "You have read Test_Title by Test_Author". It should say the opposite.

Reply:
Got it. Now the read status is checked first before giving a message.